app/wwwroot/js/coa.js

```diff
@@ -111,11 +114,12 @@ $(document).ready(function () {
     };
 
     $('#Appeal_FactumFiled input[type="radio"]').click(toggleAppeal);
+    $('#Appeal_FactumFiled input[type="radio"]').click(toggleAppeal);
```

Comment:
Is this meant to call `toggleAppeal` twice?

Reply:
Good catch. I did not mean to call that twice.  I meant to call toggleAppeal when the dropdown value was changed but it seems to work okay without it, so I'll just delete the line.